example/lstm_master.py

```diff
@@ -1,6 +1,9 @@
 import numpy
 import time
 
+import sys
+sys.path.append('..')
```

Comment:
We can probably run them from elsewhere, as long as channel.py is in the path.  But for convenience we add '..' to the path so that they can be run from their directory.

At least that is my reasoning.